inst/css/BS4/pkgdown.sass

```diff
@@ -413,12 +407,15 @@ mark {
 }
 
 .algolia-autocomplete .aa-hint {
-  color: #999;
+  color: mix($body-color, $body-bg, 90%);
 }
+
+
+/* "Max" needed until libsass supports https://sass-lang.com/documentation/syntax/special-functions#min-and-max */
 .algolia-autocomplete .aa-dropdown-menu {
-  width: max(100%, 20rem);
-  background-color: #fff;
-  border: 1px solid var(--gray);
+  width: Max(100%, 20rem);
```

Comment:
Why the capital `Max()`? Is it somehow different from `max()`?

Reply:
Answered in https://github.com/rstudio/sass/issues/92 (slowly getting to your feedback, very useful, thanks a ton already)

Reply:
Oh interesting, thanks!